.gitpod.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM gitpod/workspace-mysql
```

Comment:
```suggestion
FROM maven:3.6.3-jdk-8-openj9
```

Reply:
Just Maven with the OpenJDK should be enough. https://www.gitpod.io/docs/config-docker/